resources/github-actions.md

```diff
@@ -123,45 +123,59 @@ As an example, `rapids-print-env` is used to print common environment informatio
 
 ## Downloading CI Artifacts
 
-For NVIDIA employees with VPN access, artifacts from both pull-requests and branch builds can be accessed on [https://downloads.rapids.ai/](https://downloads.rapids.ai/).
+Artifacts from both pull-requests and branch builds can be accessed on the Github UI for the specific workflow run, found in the Actions tab of the repository.
 
-There is a link provided at the end of every C++ and Python build job where the build artifacts for that particular workflow run can be accessed.
+![](/assets/images/workflow-ui.png)
 
-![](/assets/images/downloads.png)
+There is also a link provided in the `Run actions/upload-artifact@v4` step of every C++ and Python build job where the build artifacts for that particular job can be accessed.
+
+![](/assets/images/downloads-github.png)
+
+This link downloads the required artifact as a zip file.
+
+This can also be done using `ghcli` using the following command to download the artifact. This command also unzips the artifact to the destination location:
```

Comment:
```suggestion
This can also be done using the following `gh` CLI command to download the artifact. This command also unzips the artifact to the destination location:
```